docs/tutorials/tabular_prediction/tabular-faq.md

```diff
@@ -139,6 +139,17 @@ task.fit(..., excluded_model_types=['KNN','RF','XT'])
 
 To learn how to add your own custom models to the set that AutoGluon trains, tunes, and ensembles, review :ref:`sec_tabularcustommodel`.
 
+### How can I reproduce the results of model XYZ without using AutoGluon?
+
+Exactly reproducing the results of AutoGluon models without using AutoGluon is extremely challenging.
```

Comment:
`extremely challenging` -> `can be challenging`.

Reply:
Updated